common-npm-packages/azure-arm-rest/azure-arm-app-service-kudu.ts

```diff
@@ -528,6 +528,41 @@ export class Kudu {
         }
     }
 
+    public async oneDeploy(webPackage: string, queryParameters?: Array<string>): Promise<any> {
+        let httpRequest = new webClient.WebRequest();
+        httpRequest.method = 'POST';
+        httpRequest.uri = this._client.getRequestUri(`/api/publish`, queryParameters);
```

Comment:
hi @dannysongg  I could see the checks are failing due to a build failure. i could see the wrong reference name used here. can you change **this_client** to **this.client**